org/newEigenBeta.ts

```diff
@@ -0,0 +1,113 @@
+import { Create } from "github-webhook-event-types"
+import { danger } from "danger"
+
+import * as semverSort from "semver-sort"
+import * as JiraApi from "jira-client"
+import { flatten } from "lodash"
+
+import * as IssueJSON from "../fixtures/jira_issue_example.json"
+type Issue = typeof IssueJSON
+
+const companyPrefix = "artsyproduct"
+
+// Implementation borrowed from https://github.com/danger/peril-settings/blob/b306662628f1a4c5c4ebc5f0531e15561fe60fa5/org/new_tag.ts#L12-L64
+export default async (create: Create) => {
+  if (create.ref_type !== "tag") {
+    return console.log("Skipping because it's not a tag")
+  }
```

Comment:
We should probably also skip tags that have the `-submission` suffix, since they aren't betas.